src/index.js

```diff
@@ -100,6 +120,12 @@ export function filterCards(cards, options) {
 }
 
 export function deckFromCards(cards, images, options) {
+	// load syntax-highlighting css
+	let css = fs.readFileSync(path.resolve(__dirname, `../node_modules/@highlightjs/cdn-assets/styles/${options.codeStyle}.min.css`),'utf8');
+	// remove comment from css, because that breaks things for some reason
+	css = css.replace(/\/\*[\s\S]*\*\//gm, '');
+	// load some more default css styles
+	css += fs.readFileSync(path.resolve(__dirname, `./style.css`),'utf8');
```

Comment:
@RvNovae I think this should work now. Feel free to look over the regex, I just hacked that together until it seemed to work 😅

Reply:
Alrighty, thank you!
I'll take a closer look at this later.